masterlist.yaml

```
@@ -4767,6 +4767,25 @@ plugins:
       # ELFX
       - crc: 0xAD111166
         util: 'FO4Edit v4.1.5k'
+
+  - name: 'Rebuild_Settlements.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/79618' ]
```

Closing slash for link.
Useful to have them in cases where you want to change links with search and replace.

```suggestion
    url: [ 'https://www.nexusmods.com/fallout4/mods/79618/' ]
```

---

Fixed in 22880362f1ab48beb6616916ade672ce643ea8c9